app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SessionsController < ApplicationController
+  def new
+    render :new
+  end
+
+  def create
+    user_info = request.env['omniauth.auth']
```

Comment:
não precisa de certa forma colocar alguma info do user na sessão? Sugestão: substituir o "Hello World" por "Hello, Guifs" pegando a info da sessão. desta forma nos "força" a evoluir esta parte do callback para oq for necessário. 

faz sentido?

Reply:
Faz sentido sim, pensei em elaborar mais essa parte para a inclusão do devise, mas para finalidade de aprimorar conhecimentos, acredito que faça sentido, Vou implementar aqui

Reply:
ah blz, se isso ja tava no teu plano, pode ignorar meu comentario. de qq forma, segue como achar pertinente, boa valeu